### PR TITLE
Update bug report template

### PR DESCRIPTION
**Pull request by stefanprodan**
Add explicit steps for reproducing the bug

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,24 +8,26 @@ assignees: ''
...
 **Logs**
-If applicable, please provide logs of `fluxd` or the helm-operator. In a standard stand-alone installation of Flux, you'd get this by running `kubectl logs -n default deploy/flux`.
+
+If applicable, please provide logs of `fluxd`. In a standard stand-alone installation of Flux, you'd get this by running `kubectl logs deploy/flux  -n flux`.
 
 **Additional context**
-Add any other context about the problem here, e.g 
+
 - Flux version: 
 - Helm Operator version:
```
Comment (hiddeco) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
```

Comment (stefanprodan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I left this on purpose for those that report image updates issues in HelmReleases

Comment (2opremio) [reply .github/ISSUE_TEMPLATE/bug_report.md]: That's a good one

Comment (hiddeco) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ack. Technically speaking image updates have little to do with the Helm operator version people are running though. The custom resource is selected based on the top level metadata and all other properties have no impact on the functionalities of Flux.

**Pull request merged**